### DISCUSSION TITLE
posts/2020-02-06-how-to-calculate-contrasts-from-a-fitted-brms-model/

### DISCUSSION THREAD

**Discussion by giscus** [mvuorre/mvuorre.github.io#40] [giscus]
# posts/2020-02-06-how-to-calculate-contrasts-from-a-fitted-brms-model/

Answer more questions with your estimated parameters, without refitting the model.

https://mvuorre.github.io/posts/2020-02-06-how-to-calculate-contrasts-from-a-fitted-brms-model/

**Comment by algaebrown** [comment]
Thank you Matti. This is very helpful post! Charlene
